controllers/team.controller.js

```diff
@@ -12,56 +12,34 @@ const Constants = {
 };
 
 /**
- * @async
- * @function findById
- * @param {{body: {id: ObjectId}}} req 
- * @param {*} res 
- * @return {JSON} Success or error status
- * @description Finds a team by it's mongoId that's specified in req.param.id in route parameters. The id is moved to req.body.id from req.params.id by validation.
+ * @function showTeam
+ * @param {{body: {team: Object}}} req
+ * @param {*} res
+ * @return {JSON} Success status and team object
+ * @description Returns the JSON of team object located in req.body.team
  */
-async function findById(req, res) {
-    const team = await Services.Team.findById(req.body.id);
-
-    if (team) {
-        return res.status(200).json({
-            message: Constants.Success.TEAM_GET_BY_ID,
-            data: team.toJSON()
-        });
-    } else {
-        return res.status(404).json({
-            message: Constants.Error.TEAM_404_MESSAGE,
-            data: {}
-        });
-    }
+async function showTeam(req, res) {
```

Comment:
consider changing to "gotTeam" to follow other controllers?

Reply:
I don't think I understand. The other controllers follow 'show<Type>' format, such as showAccount, showHacker, and showSponsor.

Reply:
Hmm. We might want to change those as well then, if there is no semantic difference between showX and gotX.

Reply:
Why would we change it to gotX? Is there a naming convention for controllers?

Reply:
Well it’s just that we should maintain unity of some sort— whether we want them all to be “showX” or “gotX”. I was thinking “gotX” was nice because it comes after the “getX” middleware, and is executed only once we actually got X. If you want, shelve this and make an issue for refactoring names.

Reply:
Oh I see, lots of the other controllers have the past-tense-X format. I'll make an issue about it - I'd like to have a chat about the naming convention.